src/components/card/index.tsx

```diff
@@ -87,15 +92,28 @@ export const Card = (props: ICardProps) => {
         alt={'logo'}
       />
       <div style={{ ...defaultTextParentStyles }}>
-        <text style={{ ...defaultTitleStyles, ...titleStyle }}>{title}</text>
-        <br></br>
-        <text style={{ ...defaultTextStyles, ...textStyle }}>{text}</text>
+        <text
+          style={{ ...defaultTitleStyles, ...titleStyle, display: 'block' }}
+        >
+          {title}
+        </text>
+        <text style={{ ...defaultTextStyles, ...textStyle, display: 'block' }}>
+          {text}
+        </text>
       </div>
-      <div>
+      <div style={cardButtons}>
         {primaryBtnLabel ? (
           <button
             disabled={disablePrimaryBtn}
-            style={{ ...defaultButtonStyles, ...primaryBtnStyle }}
+            style={
+              disablePrimaryBtn
+                ? {
+                    ...defaultButtonStyles,
+                    ...primaryBtnStyle,
+                    color: 'grey'
```

Comment:
Think we should allow users to pass their own color here while we maintain the default `grey`.

Reply:
set different style for disabled button.